src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,47 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  checkHealth() {
+    if (this.health <= 0) {
+      this.die();
+    }
+  }
```

Comment:
The `checkHealth` method is redundant since the `die` method is already filtering out dead animals from the `Animal.alive` array. You can remove this method and call `die` directly within the `bite` method after decreasing health.